tests/php/test-class-amp-reader-themes-rest-controller.php

```diff
@@ -96,4 +97,48 @@ public function test_get_items() {
 		$this->assertEquals( [ 'my-theme', 'legacy' ], wp_list_pluck( $data, 'slug' ) );
 		remove_filter( 'amp_reader_themes', $filter );
 	}
+
+	/**
+	 * Test the REST Response headers when themes_api was successful.
+	 *
+	 * @covers ::get_items
+	 */
+	public function test_get_items_header_with_themes_api_success() {
+		$response = $this->controller->get_items( new WP_REST_Request( 'GET', 'amp/v1' ) );
+		$this->assertEquals( [], $response->get_headers() );
+	}
+
+	/**
+	 * Tests the REST response headers when themes_api fails.
+	 *
+	 * @covers ::get_items
+	 */
+	public function test_get_items_header_with_themes_api_failure() {
+		add_filter( 'themes_api_result', '__return_null' );
+
+		$response = $this->controller->get_items( new WP_REST_Request( 'GET', 'amp/v1' ) );
+		$this->assertEquals(
+			[ 'X-AMP-Theme-API-Error' => 'The request for reader themes from the WordPress.org resulted in an invalid response. Please try again later or contact your host.' ],
```

Comment:
```suggestion
			[ 'X-AMP-Theme-API-Error' => 'The request for reader themes from the WordPress.org resulted in an invalid response. Check your Site Health to confirm that your site can communicate with WordPress.org. Otherwise, please try again later or contact your host.' ],
```